crypto/provider_core.c

```diff
@@ -893,8 +893,8 @@ static const OSSL_DISPATCH core_dispatch_[] = {
     { OSSL_FUNC_CORE_SET_ERROR_MARK, (void (*)(void))core_set_error_mark },
     { OSSL_FUNC_CORE_CLEAR_LAST_ERROR_MARK,
       (void (*)(void))core_clear_last_error_mark },
-    { OSSL_FUNC_CORE_POP_ERROR_TO_MARK,
-      (void (*)(void))core_pop_error_to_mark },
+    { OSSL_FUNC_CORE_POP_ERROR_TO_MARK, (void (*)(void))core_pop_error_to_mark },
+    { OSSL_FUNC_CORE_VSNPRINTF, (void (*)(void))BIO_vsnprintf },
```

Comment:
Mind moving this line down a bit, so it's together with `OSSL_FUNC_BIO_VPRINTF`?

Reply:
No I dont mind - but it doesnt have too much to do with BIO's

Reply:
It's in the BIO API...  but otherwise, you're remark is quite right.  However, they're the same kind of function, so...
(this is like discussing that while fprintf is a FILE function, printf isn't, and yet, they are usually presented together)